src/store/error-store.js

```diff
@@ -0,0 +1,22 @@
+import { Store } from 'consus-core/flux';
+
+let error = null;
+
+class ErrorStore extends Store{
```

Comment:
could have sworn I brought this up when I started making the error store.. so do you want me to remove it?